index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="/main.css" type="text/css"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> 
+    <title>Alef's Portfolio</title>
+</head>
+<body>
+
+    <header>
+        <a href="">
+            <img src="./assets/svg/Logo-Placeholder.svg" alt="site logo">
+        </a>
+        <a href="#" class="header--menu--icon">
+            <img src="/assets/svg/Icon-Menu.svg" alt="menu icon">
+        </a>
+    </header>
+    <main>
+        <section class="headline--container">
+            <div class="headline--title">
+                <h2>Aléf Garrido</h2>
+                <h1>Software Developer</h1>
+                <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+            </div>
+            <span class="devider">
+                <img src="/assets/svg/Devider.svg" alt="Section devider">
+            </span>
+            <section class="skills--container">
+                <div class="skills--item">
+                    <h3>Language</h3>
+                    <p>Javascript</p>
+                    <p>Ruby</p>
+                    <p>Html</p>
+                    <p>Css</p>
+                </div>
+                <div class="skills--item">
+                    <h3>Frameworks</h3>
+                    <p>Bootstrap</p>
+                    <p>Ruby on Rails</p>
+                    <p>RSPec</p>
+                    <p>CapyBara</p>
+                    <p>Selenium</p>
```

Comment:
I suggest you wrap this section using the `ul` tag to group them as it is best practice